Makefile

```diff
@@ -121,6 +121,15 @@ binary:
 	@if [ -z "${COMPONENT}" ]; then echo "Try one of the following targets instead:" \
 		&& for b in binaries ${BINARIES}; do echo "- $$b"; done && exit 1; fi
 	mkdir -p release/${TAG}
+	rm -f cmd/${COMPONENT}/resource.syso
+	if [ "${GOARCH}" = "amd64" ]; then sixtyfour="-64"; fi; \
+	goversioninfo $$sixtyfour -o cmd/${COMPONENT}/resource.syso \
+	-original-name ${COMPONENT}_${GOOS}_${GOARCH}${FILEEXT} \
+	-trademark "${COMPONENT} is a Trademark of Storj Labs, Inc." \
+	-description "${COMPONENT} program for Storj" \
+	-product-ver-build 2 -ver-build 2 \
+	-product-version "alpha2" \
```

Comment:
These three bits of version will need to be abstracted out into a variable at some point.